.github/workflows/release.yml

```diff
@@ -241,7 +241,7 @@ jobs:
       CONCURRENCY: ${{ github.ref_name }}
       PY_DOCS_VERSION_NAME: ${{ inputs.release-type == 'final' && needs.version.outputs.final || 'dev' }}
       CPP_DOCS_VERSION_NAME: ${{ inputs.release-type == 'final' && 'stable' || 'dev' }}
```

Comment:
Does it make sense to change CPP too?

Reply:
For C++ we upload the docs twice, once to named (the `stable`/`dev` one) and once to versioned (`0.24.0`). I don't know why we do it that way, I'm pretty sure I just replicated how it used to be on gh-pages :thinking: either way it works so I think it's fine to leave it as is:
https://ref.rerun.io/docs/cpp/stable/structrerun_1_1archetypes_1_1VideoStream.html